rocAL/rocAL/include/augmentations/color_augmentations/node_brightness.h

```diff
@@ -29,7 +29,7 @@ THE SOFTWARE.
 class BrightnessNode : public Node
 {
 public:
-    BrightnessNode(const std::vector<rocalTensor *> &inputs, const std::vector<rocalTensor *> &outputs);
+    BrightnessNode(const std::vector<Tensor *> &inputs, const std::vector<Tensor *> &outputs);
```

Comment:
why is this renamed?

Reply:
In general we do not have the prefix rocal to any of the internal functions or the class/struct. Since we wanted to expose the class externally we had named it as rocalTensor and rocalTensorList initially. With this PR we will change all instances internally from Image->Tensor and ImageInfo->TensorInfo and the external API exposed to the user will be rocalTensor and rocalTensorList.
This will be just a one time change and we can follow this convention for upcoming PR.